backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/storage/impl/ClassAttributeStorageFactory.java

```diff
@@ -159,7 +156,7 @@ public IAttributeStorage getAttributeStorageIfHandled(final Object model)
 
 	@Override
 	public final IHUAttributesDAO getHUAttributesDAO()
-	{
+	{     // TODO tbp: this exception is false: this.huAttributesDAO is NOT NULL!
```

Comment:
let's leave the TODO here, until we get to check out the code and better understand why we are really throwing the exception